driver/src/main/java/org/neo4j/driver/v1/Config.java

```diff
@@ -241,32 +241,44 @@ public Config toConfig()
     }
 
     /**
-     * A configuration to configure TLS authentication
+     * Control the level of encryption to require
+     */
+    public enum EncryptionLevel
+    {
+        /** With this level, the driver will only connect to the server if it can do it without encryption. */
+        REJECTED,
```

Comment:
+1. BTW, libneo4j-client handles this with a single `INSECURE` flag, which is essentially a Boolean, since there is only two different "levels" anyway.